File: 10.4_MyPhotos/app/src/main/java/com/example/a55myphotos/View/SinglePhotoActivity.java
package com.example.a55myphotos.View;

import androidx.appcompat.app.AppCompatActivity;
import android.content.Intent;
import android.os.Bundle;
import android.widget.ImageView;
import android.widget.TextView;
import com.example.a55myphotos.MainActivity;
import com.example.a55myphotos.R;
import com.example.a55myphotos.model.Photo;
import com.squareup.picasso.Picasso;
import java.util.List;

public class SinglePhotoActivity extends AppCompatActivity {
    private ImageView imageView;
    private TextView textView_title;
    private Photo photo;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_single_photo);
        imageView = findViewById(R.id.single_image);
        textView_title = findViewById(R.id.single);
        photo = (Photo) getIntent().getSerializableExtra("MyObject");
        textView_title.setText(photo.getTitle());
        Picasso.get()
                .load(photo.getUrl())
                .into(imageView);
    }
}

File: MYPHOTOS_Kotlin/app/src/main/java/com/example/myphotos_kotlin/View/SinglePhotoActivity.kt
package com.example.myphotos_kotlin.View

import android.os.Bundle
import android.widget.ImageView
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import com.example.myphotos_kotlin.MainActivity
import com.example.myphotos_kotlin.R
import com.example.myphotos_kotlin.model.Photo
import com.squareup.picasso.Picasso

class SinglePhotoActivity : AppCompatActivity() {
    lateinit var imageView: ImageView
    lateinit var textView: TextView
    lateinit var photo: Photo
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_single_photo)
        textView = findViewById(R.id.single)
        imageView = findViewById(R.id.single_image)
        val intenet = intent

        var photo = intenet.getParcelableExtra<Photo>(MainActivity.KEY_PHOTO_BUNDLE)
        //Implement object from Bundle
        textView.text = photo?.title


        if (photo != null) {
            Picasso.get()

                .load(photo.url)
                .into(imageView)
        }

    }
}